CHANGELOG.rst

```diff
@@ -44,7 +44,9 @@
 
 * Updated docstrings with a more uniform style.
 
-*
+* Testcase for overriding autouse fixtures with a parametrized one.
+  Thanks to `@astraw38`_ for reporting the issue (`#1601`_)
+  Thanks to `@diegorusso`_ for the PR
```

Comment:
I've dropped the changes to `CHANGELOG.rst` file and squashed two commits in one